spec/fixtures/hourlies/date_008.marc

```diff
@@ -0,0 +1 @@
+02067cam a2200469 a 4500001000800000005001700008008004100025010001700066020001500083020001800098029002100116029001900137029001700156035001200173035001200185035001600197035002000213040006400233049000900297050002200306082002100328084001500349100002200364245014800386246002600534260003900560300002700599500005800626504006600684505032800750650002701078650003101105700001901136856009901155856009101254910002601345938007101371938004001442938003901482991006401521994001201585271018320080307152200.0010831s2002    nyu      b    001 0 eng    a  2001050014  a0375714499  a97803757144981 aNLGGCb2461901591 aYDXCPb18130101 aNZ1b6504593  a2710183  a2710183  aocm47971712  a(OCoLC)47971712  aDLCcDLCdUSXdBAKERdNLGGCdNPLdYDXCPdOCLCQdBTCTAdMdBJ  aJHEE00aP96.E25bH47 200200a381/.4530223221  a05.302bcl1 aHerman, Edward S.10aManufacturing consent :bthe political economy of the mass media /cEdward S. Herman and Noam Chomsky ; with a new introduction by the authors.14aManugacturing content  aNew York :bPantheon Books,c2002.  alxiv, 412 p. ;c24 cm.  aUpdated ed. of: Manufacturing consent. 1st ed. c1988.  aIncludes bibliographical references (p. [331]-393) and index.0 aA propaganda model -- Worthy and unworthy victims -- Legitimizing versus meaningless third world elections: El Salvador, Guatemala, and Nicaragua -- The KGB-Bulgarian plot to kill the Pope: free-market disinformation as "news" -- The Indochina wars (I): Vietnam -- The Indochina wars (II): Laos and Cambodia -- Conclusions. 0aMass mediaxOwnership. 0aMass media and propaganda.1 aChomsky, Noam.423Contributor biographical informationuhttp://www.loc.gov/catdir/bios/random051/2001050014.html423Publisher descriptionuhttp://www.loc.gov/catdir/description/random044/2001050014.html  a2710183bHorizon bib#  aBaker & TaylorbBKTYc18.95d14.21i0375714499n0003788716sactive  aYBP Library ServicesbYANKn1813010  aBaker and TaylorbBTCPn2001050014  aP96.E25 H47 2002flcbelc1cc. 1q0i4659750lembluememsel  aC0bJHE
```

Comment:
I don't see you loading these files anywhere. Am I missing something?

Reply:
These get loaded here. (https://github.com/psu-libraries/psulib_traject/blob/a8cd1765848ffec023ac5033e7a3450437bfe48f/spec/lib/psulib_traject/workers/hourly_indexer_spec.rb)

The tests ensure that both marc, and mrc files are read, and that exactly 2 jobs are put on the queue.